site/doc/config/reference.md

```diff
@@ -254,6 +255,41 @@ In this case the run step name will be the same of the command trimmed at the ma
 | environment | Map: EnvVarName(String) => EnvVarValue([Value](#value)) | Environment variables to set                                                       |
 | working_dir | String                                                  | Working dir where the entrypoint will be executed (also used for steps in a task)  |
 | user        | String                                                  | The user id or username to use when executing the entrypoint or the task run steps |
+| volumes     | List: [Volume](#volume)                                 | A list of volumes (currently only `tmpfs` is supported)                            |
+
+###### Volume
+
+Containers can mount one or more volumes (currently only `tmpfs` is supported).
+
+| Option | Type                           | Description  |
+| -------| ------------------------------ | ------------ |
+| path   | String                         | Mountpoint   |
+| tmpfs  | [TmpFS Volume](#tmpfs-volume)  | tmpfs volume |
+
+Example of single volume with size limit:
+```yaml
+  - image: image01
+    volumes:
+      - path: /mnt/tmpfs
+        tmpfs:
+          size: 1Gi
+```
+Example with two volumes, the former with a size limit and the latter without (so the maximum size will be managed by the underlying OS):
```

Comment:
`(so the maximum size will be managed by the underlying OS)`

I'll remove this statement since there are many variables and not only the OS that are unknown or will change (they depend on how the executor: docker or k8s behave and how it's configured). I'm not sure how this could be written...